README.md

```diff
@@ -1,4 +1,262 @@
+
+Some changes do not require an RFC:
+
+  - Rephrasing, reorganizing, refactoring, or otherwise "changing shape does not change meaning".
+  - Additions that strictly improve objective, numerical quality criteria
+    (warning removal, speedup, better platform coverage, more parallelism, trap more errors, etc.)
+  - Additions only likely to be _noticed by_ other developers-of-ipfs, but
+    invisible to users-of-ipfs.
+
+If you submit a pull request to implement a new feature without going through
+the RFC process, it may be closed with a polite request to submit an RFC first.
+
+
+### working group specific guidelines
+[working group specific guidelines]: #working group-specific-guidelines
+
+As working groups take form, they will have the prerogative to add further guidelines regarding RFCs that apply to their domain of work.
+
+## Before creating an RFC
+[Before creating an RFC]: #before-creating-an-rfc
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality
+proposals, proposals for previously-rejected features, or those that don't fit
+into the near-term roadmap, may be quickly rejected, which can be demotivating
+for the unprepared contributor. Laying some groundwork ahead of the RFC can
+make the process smoother.
+
+Although there is no single way to prepare for submitting an RFC, it is
+generally a good idea to pursue feedback from other project developers
+beforehand, to ascertain that the RFC may be desirable; having a consistent
+impact on the project requires concerted effort toward consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking
+the idea over on the #ipfs irc channel, filing and discussing ideas on the
+[RFC issue tracker], and occasionally posting "pre-RFCs" on the
+[ipfs discussion forum](https://discuss.ipfs.io) for early review.
+
+As a rule of thumb, receiving encouraging feedback from long-standing project
+developers, and particularly members of the relevant [working group] is a good
+indication that the RFC is worth pursuing.
+
+
+## The RFC life-cycle
+[The RFC life-cycle]: #the-rfc-life-cycle
+
+In short, to get a major feature added to IPFS, one must first get the RFC
+merged into the RFC repository as a markdown file. At that point the RFC is
+"Accepted" and may be implemented with the goal of eventual inclusion into the corresponding protocols or libraries.
+
+As it grows from an idea to shipping in a supported release, an SDK RFC will traverse along:
+
+1. **Draft**: The owner of the SDK RFC has started to draft up how the subject will be handled and may be reviewing with a core group.  Comments are certainly welcome at this stage even though the owner hasn't worked through enough details to ask for...
+2. **Review**: This SDK RFC is in a review period. Stakeholders and the owner may still be iterating on some final details before signoff. A minimum review period has been defined.
```

Comment:
> A minimum review period has been defined.

Defined.. where? I'd make this more precise, eg. _defined in the PR._

Reply:
I'm just taking that sentence out because it's not clear. If someone wants to propose a minimum review period we can add it back in.